server/models/bookings.js

```diff
@@ -47,12 +58,18 @@ export default class Book {
     return newTrip.findTripDate(Date);
   }
 
+  async deleteBookingByUser(booking_id, user_id){
+    const result = await db.deleteBooking(booking_id, user_id);
+    return result;
+  }
+
   async checkBookingUser(userid) {
     // return this.getAtllbookings.find(x => x.userId === userid);
```

Comment:
What's the purpose of this method? If it doesn't do anything please remove it

Reply:
I am going to use it when am getting user bookings